students/brian_warn/session03/mailroom.py

```diff
@@ -0,0 +1,112 @@
+#!/Library/Frameworks/Python.framework/Versions/3.6/bin/python
+# mailroom1.py
+# Display summary donor information: name, number of gifts, total amount for all gifts, average gift amount
+
+# Donor names
+# Note: Done with dictionary because I couldn't figure out how to do this using lists.
+donor_gifts = {
+    'William Gates': [100000, 23456, 891234],
+    'Jeff Bezos': [654321, 56754],
+    'Oscar Muñoz': [908765, 8796],
+    'Mark Zuckerberg': [2000000]
+}
+
+
+def main():
+# thank_you -->
+    def thank_you(donor_gifts):
```

Comment:
you don't want to create these functions inside the main() function.

create them in the module namespace, and then call them from main()